src/test/java/com/musseukpeople/woorimap/post/presentation/PostControllerTest.java

```diff
@@ -0,0 +1,64 @@
+package com.musseukpeople.woorimap.post.presentation;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.*;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockHttpServletResponse;
+
+import com.musseukpeople.woorimap.auth.application.dto.request.SignInRequest;
+import com.musseukpeople.woorimap.tag.application.dto.TagRequest;
+import com.musseukpeople.woorimap.member.application.dto.request.SignupRequest;
+import com.musseukpeople.woorimap.post.application.dto.CreatePostRequest;
+import com.musseukpeople.woorimap.util.AcceptanceTest;
+
+public class PostControllerTest extends AcceptanceTest {
+
+    private String coupleAccessToken;
+
+    @BeforeEach
+    void login() throws Exception {
+        String email = "test@gmail.com";
+        String password = "!Test1234";
+        String nickName = "test";
+        SignupRequest user = new SignupRequest(email, password, nickName);
+        String accessToken = 회원가입_토큰(user);
+        커플_맺기(accessToken);
+        coupleAccessToken = 로그인_토큰(new SignInRequest(user.getEmail(), user.getPassword()));
```

Comment:
```suggestion
        coupleAccessToken = 커플_맺기_토큰(accessToken);
```

AcceptanceTest에 있어서 커플 맺고 다시 로그인 하지 않아도 되는 귀찮음을 제거했습니다 ㅎㅎ

Reply:
오 굿